contracts/cw20-atomic-swap/src/contract.rs

```diff
@@ -31,22 +37,48 @@ pub fn handle<S: Storage, A: Api, Q: Querier>(
     msg: HandleMsg,
 ) -> StdResult<HandleResponse> {
     match msg {
-        HandleMsg::Create(msg) => try_create(deps, env, msg),
+        HandleMsg::Create(msg) => try_create(
+            deps,
+            env.clone(),
+            msg,
+            Balance::Native(env.message.sent_funds),
```

Comment:
This is interesting... so `try_create` works for both native and cw20 coins.
It would be good to write some small docs explaining this pattern (for the docs site - talk with @orkunkl )

Reply:
OK.